tcms/testruns/models.py

```diff
@@ -103,8 +102,8 @@ def add_case_run(self, case, status=1, assignee=None,
             _case_text_version = case.history.latest().history_id
 
         _assignee = assignee \
-            or (case.default_tester_id and case.default_tester) \
-            or (self.default_tester_id and self.default_tester)
+                    or (case.default_tester_id and case.default_tester) \
+                    or (self.default_tester_id and self.default_tester)
```

Comment:
flake8

Reply:
why do you have to change this ?

Reply:
Again, my IDE is trying to be clever. Will spend some time configuring it after finishing this one